### ISSUE TITLE
Add title property for Toolbox tools

### ISSUE THREAD

**Issue by gohabereg** [codex-team/codex.editor#498]
Property value should be used as alt text for toolbox icons

![image](https://user-images.githubusercontent.com/23050529/48952741-4e045c00-ef54-11e8-8043-8728f07496c3.png)

**Comment by gohabereg** [comment]
resolved #540
